Create ui-c8y-1023-14-48-slow-performance-in-dashboard.md (#4159)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-14-48-slow-performance-in-dashboard.md

---
date: ""
title: Fixed issue with longer dashboard loading time
product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-5463
version: 1023.14.48
---

> Due to some recent changes related to rolling out Dashboard manager in Device management some users reported longer loading times for dashboard views. This issue has been address and dashboard loading times have returned to normal.

Reviewer: ```suggestion
Due to some recent changes related to rolling out the dashboard manager in the Device Management application, the loading time for dashboard views increased. This issue has been addressed, and dashboard loading times have returned to normal.
```

---
date: ""

> title: Slow performance in dashboard

Reviewer 2: ```suggestion
title: Fixed issue with longer dashboard loading time
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-5463
version: 1023.14.48
---
# Backport

This will backport the following commits from `develop` to
`release/y2026`:
- [fix(Web SDK): [DM-5463] Slow performance in dashboard
(#10846)](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/10846)

<!--- Backport version: 9.5.1 -->

### Questions ?
Please refer to the [Backport tool
documentation](https://github.com/sorenlouv/backport)

[DM-5463]:
https://cumulocity.atlassian.net/browse/DM-5463?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

> Co-authored-by: Bozhidar Dedov <95280800+bozhidardedov@users.noreply.github.com>

Reviewer 2: ```suggestion
Due to some recent changes related to rolling out Dashboard manager in Device management some users reported longer loading times for dashboard views. This issue has been address and dashboard loading times have returned to normal.
```
